packages/backend/src/config/app.ts

```diff
@@ -61,6 +61,8 @@ export const Config = {
   accountWorkers: envInt('ACCOUNT_WORKERS', 1),
   accountWorkerIdle: envInt('ACCOUNT_WORKER_IDLE', 200), // milliseconds
 
+  authServer: envString('AUTH_SERVER', 'http://127.0.0.1:3006'),
```

Comment:
I think we're going to end up with two urls for the auth server:
- one for grant requests (this)
- one for token introspection

https://datatracker.ietf.org/doc/html/draft-ietf-gnap-resource-servers#section-3.1